### ISSUE TITLE
Support for remote signer?

### ISSUE THREAD

**Issue by JakkuSakura** [baking-bad/pytezos#301]
I'm integrating pytezos with a HSM, which private keys are locked within a dedicated hardware.

However, I didn't see explicit support for it in pytezos. I'm reimplementing pytezos.Key, where signing is done with query to HSM. Am I on the right track?

**Comment by m-kus** [comment]
Yeah, you can inherit from `pytezos.Key` indeed and override `sign`/`verify` methods
